images/build/installDotNetCore.sh

```diff
@@ -24,9 +24,9 @@ if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]; then
     set -x
     
     if [ $isSasTokenEmpty -eq 0 ]; then
-      echo "sasToken cannot be empty for $sdkStorageAccountUrl."
+        echo "sasToken cannot be empty for $sdkStorageAccountUrl."
     else
-        echo "sasToken is valid for $sdkStorageAccountUrl."
+        echo "sasToken is empty for $sdkStorageAccountUrl."
```

Comment:
Is this correct? `isSasTokenEmpty` is set to 0 on line 22 if the value of `sasToken` provided is empty, so the above if-statement is correct and this `else` statement should only be hit if `sasToken` has some value.